Python/bytecodes.c

```diff
@@ -3976,13 +3984,14 @@ dummy_func(
 // END BYTECODES //
 
     }
- error:;
- exception_unwind:;
- handle_eval_breaker:;
- resume_frame:;
- resume_with_error:;
- start_frame:;
- unbound_local_error:;
+ error:
+ exception_unwind:
+ handle_eval_breaker:
+ resume_frame:
+ resume_with_error:
+ start_frame:
+ unbound_local_error:
+    ;
```

Comment:
Is this semicolon really needed? My IDE (VS Code) doesn't complain...

Reply:
It's an error to real C compilers, so better safe than sorry.